### PR TITLE
F/home

### PR DESCRIPTION
**Pull request by gozman**

### CODE REVIEW

File: pages/docs/index.md [hunk 1]
```diff
@@ -1,7 +1,72 @@
...
+ * Keep multiple stakeholders in the loop by ensuring that messages sent from a customer to your business are mirrored across the different tools in use in your business.
+ * Add bots to your customer success arsenal in order to automate common tasks - while keeping a human in the loop in the conversation to supervise the bot's communication.
+
+## Which messaging channels are supported?
+
+Smooch is committed to supporting the most messaging channels on the market with the richest set of functionality. Today, you can use Smooch to converse on:
+
+#### **TODO** Row of logos here
+
+Web Chat, iOS SDK, Android SDK, Messenger, WeChat, Telegram, LINE, Twilio, E-Mail
```
Comment (mspensieri) [inline pages/docs/index.md]: Missing Viber here

File: pages/docs/index.md [hunk 2]
```diff
@@ -1,7 +1,72 @@
...
+
+**Business?**
+
+The [Business Quickstart](/docs/business-quickstart/) walks you through adding conversations to your favourite business system, using our quick and easy integrations.
+
+---
+
+## We're here to help!
+
+Need help? If you can't find the answer you're looking for in this documentation, get in touch with us by [email](mailto:help@smooch.io) or via your favourite messaging app [messaging app](#openMessenger).
```
Comment (mspensieri) [inline pages/docs/index.md]: "messaging app" is duplicated here.

Also where does the link go?

Comment (gozman) [inline pages/docs/index.md]: intent was to open the web messenger, but we won't do it.

File: pages/docs/index.md (hunk 1 above)
Comment (gozman) [inline pages/docs/index.md]: thx

File: pages/docs/index.md (hunk 2 above)
Comment (jpjoyal) [inline pages/docs/index.md]: suggest sending to our [help page](https://smooch.io/help/) where all the support/contact options are available.

**Pull request merged**